entity-framework/core/get-started/xamarin.md

```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
 }
 ```
 
+## iPhone Configuration
+
+Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
```

Comment:
```suggestion
Entity Framework Core sometimes generates dynamic code for performance reasons. This isn't supported on the iOS platform and may cause the app to crash. The Xamarin team built an interpreter to handle these scenarios. It should be configured in the `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
```

Reply:
Per MS style guide, prefer contractions. Also the style guide insists files be `*italic*`, but for project files, I prefer code fencing as you've done.